dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -68,10 +73,10 @@ public GHRepository getRepo(String repoName) throws IOException {
             throw new IOException("Invalid image name.");
         }
         search.q("\"FROM " + query + "\"");
-        log.debug("Searching for {}", query);
+        log.info("Searching for {}", query);
```

Comment:
change change this to info?

Reply:
Oops.. I forgot to switch it back to debug.